LanguageFeatures/Augmentation-libraries/augmented_expression_A06_t03.dart

```diff
@@ -10,18 +10,18 @@
 ///   meaning and is not allowed.
 ///
 /// @description Checks that it is a compile-time error to have an augmenting
-/// declaration that has a named parameter whose name is `augmented`.
+/// declaration that calls enum constructor with a named parameter whose name is
+/// `augmented`.
 /// @author sgrekhov22@gmail.com
 
 // SharedOptions=--enable-experiment=macros
 
 import augment 'augmented_expression_A06_t03_lib.dart';
 
 enum E {
-  e1.named(augmented: 1),
-  e2.namedRequired(augmented: 2),
-  e3.record((augmented: 3));
+  e0;
 
+  const E();
   const E.named({int augmented = 0});
   const E.namedRequired({required int augmented});
   const E.record(({int augmented}) r);
```

Comment:
The rules around using `augmented` as something which is _not_ an identifier expression are probably specified sufficiently in the feature specification, but it isn't discussed explicitly. So I created https://github.com/dart-lang/language/issues/4019 in order to initiate a clarification.

I think we should just proceed with these tests, based on the current wording in the specification. We may then need to change things in several tests if 4019 or 4009 gives rise to changes.